packages/react-select/src/components/containers.tsx

```diff
@@ -84,9 +84,10 @@ export const valueContainerCSS = <
   Group extends GroupBase<Option>
 >({
   theme: { spacing },
+  isMulti
```

Comment:
To make Prettier happy:
```suggestion
  isMulti,
```